src/index.js

```diff
@@ -380,7 +383,18 @@ class MiniCssExtractPlugin {
                   'head.appendChild(linkTag);',
                 ]),
                 '}).then(function() {',
-                Template.indent(['installedCssChunks[chunkId] = 0;']),
+                Template.indent([
+                  'installedCssChunks[chunkId] = 0;',
+                  'if(supportsPreload) {',
+                  Template.indent([
+                    'var execLinkTag = document.createElement("link");',
+                    `execLinkTag.href =  ${mainTemplate.requireFn}.p + ${linkHrefPath};`,
+                    'execLinkTag.rel = "stylesheet";',
+                    'execLinkTag.type = "text/css";',
+                    'document.body.appendChild(execLinkTag);',
```

Comment:
Hi!

It's look like breaking change, because at [line#344](https://github.com/webpack-contrib/mini-css-extract-plugin/pull/344/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R344) is checking an existing link, and prevent append new link at this case.

But now, even if the link exists on the page, we get an additional unnecessary link.

Maybe better to add extra flag, checking that [line#356](https://github.com/webpack-contrib/mini-css-extract-plugin/pull/344/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R356) already append `preload` link?

Reply:
Also, we had a problem in out internal framework with this changes, because we append `preload` and `stylesheet` links directly in document.head, with some custom `onload` logic, and after package updating, i see a broken integration test with new link tag inside body)